ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobStatus.tsx

```diff
@@ -119,9 +114,17 @@ function ReportJobStatus({
     } else if (reportStatus.runState === 'PREPARING') {
         statusIcon = <InProgressIcon title="Report run is preparing" />;
         statusText = <p>Preparing</p>;
-    } else {
+    } else if (reportStatus.runState === 'WAITING') {
         statusIcon = <PendingIcon title="Report run is waiting" />;
         statusText = <p>Waiting</p>;
+    } else {
+        statusColorClass = 'pf-u-warning-color-100';
+        statusIcon = (
+            <Tooltip content="Please contact support for more help.">
+                <ExclamationTriangleIcon title="Report run status is unavailable" />
+            </Tooltip>
+        );
+        statusText = <p>Unknown status</p>;
```

Comment:
@zhenpesky When a status doesn't match anything we have, I added this. I noticed when the status change was made on the backend, the UI was just displaying `WAITING`. So this makes it easier to tell something is wrong. Let me know if you have better wording here. Also, let me know if there is a better icon for this

Reply:
looks good